src/main/kotlin/org/jitsi/nlj/util/Bandwidth.kt

```diff
@@ -111,6 +111,20 @@ class Bandwidth(bps: Double) : Comparable<Bandwidth> {
     }
 
     override fun hashCode(): Int = bps.hashCode()
+
+    companion object {
+        fun fromString(str: String): Bandwidth {
+            val (digits, notDigits) = str.partition { it.isDigit() }
+            val amount = digits.toInt()
+            val unit = notDigits.trim()
```

Comment:
Should we throw a `toLowerCase` on here as well?

Reply:
good idea, done.